src/javascript/app/base/binary_pjax.js

```diff
@@ -166,7 +166,8 @@ const BinaryPjax = (() => {
 
         const query_params = Url.paramsHash();
         if (!query_params.anchor) {
-            $.scrollTo('body', 500);
+            // $.scrollTo('body', 500); // this jQuery function causes an issue and can be replaced by the function below
```

Comment:
The reported issue might have a different reason, as `$.scrollTo` is being used in many places.